.github/workflows/voyager-debezium-workflow.yml

```diff
@@ -65,19 +65,21 @@ jobs:
       - name: Build Debezium Server
         run: |
           sh buildDebeziumServer.sh
+          cd yb-client-cdc-stream-wrapper/
+          mvn clean package
       - name: Package Debezium Server
         run: |
+          # package yb-client-cdc-stream-wrapper.jar into debezium-server
+           mv yb-client-cdc-stream-wrapper/target/yb-client-cdc-stream-wrapper.jar  debezium-server/debezium-server-dist/target/debezium-server
```

Comment:
@priyanshi-yb You'll have to move it inside the debezium-server-1.9.5 folder since that will be considered the DEBEZIUM_DIST_DIR for fall-forward.